packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/observe/TransactionTableRow.tsx

```diff
@@ -86,7 +86,9 @@ export function TransactionTableRow({ row }: { row: Row<WatchedTransaction> }) {
                       {data.functionData?.args?.map((arg, idx) => (
                         <div key={idx} className="flex">
                           <span className="flex-shrink-0 text-xs text-white/60">arg {idx + 1}:</span>
```

Comment:
(unrelated to this change)
if we have the ABI, any way to get this to be a nicely named arg instead of "arg 1"?

Reply:
good point, I'll look into that in the upcoming "decoding inputs" PR